src/prebuilt/ast-parser-gen.cc

```diff
@@ -1,8 +1,8 @@
-/* A Bison parser, made by GNU Bison 3.0.4.  */
+/* A Bison parser, made by GNU Bison 3.0.2.  */
```

Comment:
I think I have a newer version on my laptop. It would be nice to have it built automatically, but I figure as long as the travis bot that tests the prebuilt parser passes, everything should be OK.

right? :-)